201-2-vms-loadbalancer-natrules/azuredeploy.json

```diff
@@ -223,16 +160,16 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
+      "apiVersion": "2016-09-01", 
       "type": "Microsoft.Network/loadBalancers/inboundNatRules",
```

Comment:
We don't have schemas for the latest (so nothing is flagged), but the template deploys...  I *think* this may be a convention to deal with the lack of copy support outside of the top level resource